src/bridgestan.h

```diff
@@ -262,7 +262,9 @@ int bs_log_density_gradient(const bs_model* m, bool propto, bool jacobian,
  * pointer `hessian` must have enough space to hold the Hessian.
  *
  * The gradients are computed using automatic differentiation.  the
- * Hessians are
+ * Hessians are computed using nested automatic differentiation if
+ * BRIDGESTAN_AD_HESSIAN is defined, otherwise they are computed
+ * using central finite differences.
```

Comment:
Should we add "...using central finite differences of gradient vectors at theta_unc."?

Is a the missing capitalization at the beginning of this sentence?

Reply:
I think I meant to delete that ` the`. Fixed.